setup.cfg

```diff
@@ -6,7 +6,7 @@ long_description = file: README.md
 long_description_content_type = text/markdown; variant=GFM
 keywords = quantum, physics, dynamics
 license = BSD 3-Clause License
-license_file = LICENSE.txt
+license_files = LICENSE.txt
```

Comment:
The name `license_file` is deprecated. Python 3.10 raises a warning if it is used and that causes runtime compilation of coefficient strings to fail (why pyimportx looks at the setup.cfg file is a bit of a mystery -- probably it just happens to be in the current directory when pyimportx runs setuptools or distutils).